### PR TITLE
Document new parameter `min_ipv6_addr_count` and sub sensors for wifi_info and ethernet_info

### PR DESCRIPTION
**Pull request by HeMan**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#5449
## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/network.rst [hunk 1]
```diff
@@ -14,11 +14,13 @@ networks (WiFi, Ethernet).
     # Example configuration
     network:
         enable_ipv6: true
+        min_ipv6_addr_count: 2
         
 Configuration variables:
 ------------------------
 
 - **enable_ipv6** (*Optional*, boolean): Enables IPv6 support. Defaults to ``false``.
+- **min_ipv6_addr_count** (*Optional*, integer): ESPhome considers network connected when it has gotten one IPv4 address and this amount of IPv6 addresses. Defaults to ``0`` to not hang on boot with networks with no IPv6 enabled. ``2`` could be a good value for a configuration that requires IPv6.
```
Comment (kbx81) [inline components/network.rst]:
```suggestion
- **min_ipv6_addr_count** (*Optional*, integer): ESPhome considers the network to be connected when it has one IPv4 address and this number of IPv6 addresses. Defaults to ``0`` so as to not hang on boot with networks where IPv6 is not enabled. ``2`` is typically a reasonable value for configurations requiring IPv6.
```

File: components/text_sensor/ethernet_info.rst [hunk 2]
```diff
@@ -15,13 +15,25 @@ via text sensors.
...
+          address_4:
+            name: ESP IP Address 4
 
 
 Configuration variables:
 ------------------------
 
 - **ip_address** (*Optional*): Expose the IP Address of the ESP as a text sensor. All options from
   :ref:`Text Sensor <config-text_sensor>`.
+- **address_0-address_4** (*Optional*): With IPv6 and dual stack the device get's at least two ip addresses, often more. To report all addresses the config could have up to 5 sub sensor to ip_address. All options from
```
Comment (kbx81) [inline components/text_sensor/ethernet_info.rst]:
```suggestion
- **address_0-address_4** (*Optional*): With dual stack (IPv4 and IPv6) the device will have at least two IP addresses -- often more. To report all addresses the configuration may have up to five sub-sensors. All options from
```

File: components/text_sensor/wifi_info.rst [hunk 3]
```diff
@@ -31,6 +41,8 @@ Configuration variables:
 
 - **ip_address** (*Optional*): Expose the IP Address of the ESP as a text sensor. All options from
   :ref:`Text Sensor <config-text_sensor>`.
+- **address_0-address_4** (*Optional*): With IPv6 and dual stack the device get's at least two ip addresses, often more. To report all addresses the config could have up to 5 sub sensor to ip_address. All options from
```
Comment (kbx81) [inline components/text_sensor/wifi_info.rst]:
```suggestion
- **address_0-address_4** (*Optional*): With dual stack (IPv4 and IPv6) the device will have at least two IP addresses -- often more. To report all addresses the configuration may have up to five sub-sensors. All options from
```

**Review by kbx81 — DISMISSED** [review]
Just some re-phrasing for readability. 😇

**Review by jesserockz — APPROVED** [review]

**Pull request merged**